README.md

```diff
@@ -40,7 +40,6 @@ docker run -p 8080:8080 cruise/webviz
 
 ```sh
 npm run bootstrap # install dependencies
-npm run build # build all packages
```

Comment:
I'd leave this because sometimes we'll need to build individual packages for publishing.

Reply:
It is specifically in the section "Running the static Webviz application"... If it is needed for other situations, I think those should have separate section of the README.

Reply:
Make sense. I just updated branch. Feel free to merge after the CI passes.

Reply:
Thanks @vidaaudrey. Can you please merge? (I don't have permission.)